model/src/main/kotlin/config/S3FileStorageConfiguration.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2017 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
```

Comment:
S3FileStorageConfiguration.kt is a new file so please change copyright year from 2017 to 2023

Reply:
Addressed by squashing (not the best way to do it, I agree)